README.md

```diff
@@ -74,11 +82,30 @@ For actually tracing, Tracee needs to run with sufficient capabilities:
 
 Alternatively, running as `root` or with the `--privileged` flag of Docker, is an easy way to start.
 
+#### Linux Headers
+
+In order to compile the eBPF program, Tracee needs some of the Linux kernel headers. Depending on your Linux distribution, there are different ways to obtain it, and different places they end up at.  
+
+> Note that it's important that the kernel headers match the exact version of kernel you are running. To check the current kernel version, run the command `uname -r`
+
+Here are a few examples of how to obtain Linux kernel headers in different Linux distributions:
+
+Distribution | Command | Installation location
+--- | --- | ---
+Ubuntu | apt-get install linux-headers | /usr/bin
+
+When building the eBPF object file using Tracee's `make bpf`, it will assume the headers to be found in common location: `/lib/modules/${kernel_version}/build`. If you have the headers elsewhere, you need to provide them to `make` using `KERN_BLD_PATH` and `KERN_SRC_PATH` for the build and sources appropriately. 
```

Comment:
This can be a bit confusing. Not all users are aware of what is build and sources directories (even for us it took some time to figure this out). Also, the more common thing is to have all the headers in one directory, in which case only 'KERN_HEADERS" can be used